client/src/components/LandingPage/index.js

```diff
@@ -1,19 +1,67 @@
 import React from 'react';
-import Styles from './Style';
-import { Grid, Box, Typography, Paper } from '@material-ui/core';
+import Style from './Style';
+import {
+  Grid,
+  Box,
+  Typography,
+  Paper,
+  makeStyles,
+  Button,
+  Link,
+} from '@material-ui/core';
 
 export default () => {
-  const classes = Styles();
+  const classes = Style();
   return (
     <Box component="div" p={3} width={1}>
-      <Grid container item sx={12} justify="center">
-        <Paper elevation={3} className={classes.content}>
-          <Grid item xs={12}>
-            <Typography variant="h6" color="textSecondary" align="center">
-              Billbase
-            </Typography>
+      <Grid container item xs={12} justify="center">
+        <Grid item xs={12}>
+          <Typography
+            variant="h4"
+            color="textPrimary"
+            className={classes.header}
+          >
+            Billbase
+          </Typography>
+          <Typography
+            variant="h6"
+            color="textPrimary"
+            className={classes.typographyStyle}
+          >
+            Billbase Do you want to know if you’re spending too much? Have a
+            clear ii of your bills and compare it with other users
```

Comment:
change the overview message to this
`Do you want to know if you’re spending too much? Have a clear overview of your bills and compare them with other users.`